libpius/signer.py

```diff
@@ -497,7 +497,7 @@ def import_clean_key(self, key):
     cmd = [self.gpg] + self.gpg_base_opts + self.gpg_quiet_opts + [
         '--no-default-keyring',
         '--keyring', self.tmp_keyring,
-        '--import-options', 'import-minimal',
+        '--import-options', 'import-minimal,keep-ownertrust',
```

Comment:
Ok. I really appreciate the review, thx.

The current import thru import_clean_key imports both the signer and the signee key in one pass using a single .asc. This is related to export_clean_key exporting them as a single .asc.

The current import options are applied to importing this .asc file and that makes it difficult to distinguish between how the signer and signee keys are imported into the temporary pius keyring.

On way to be able to to prevent the security risk you mention would be to export the signer and signee keys to separate .asc files instead of just one. This could be achieved by:
1. changing export_clean_key so that it stores the exports of the signer and signee key into separate .asc files instead of just one
2. changing import_clean_key so that it imports the signer key .asc with the keep-ownertrust import-option added and the signee key .asc without that additional option

Before I waste your and also my time on another pull request you're not happy with, I'd appreciate it if you could let me know if this would be acceptable to you.